ci/goreleaser/goreleaser-fips.yml

```diff
@@ -0,0 +1,294 @@
+- ids:
+    - std
+  image_templates:
+    - "tykio/tyk-gateway:{{ .Tag }}-amd64"
+    - "docker.tyk.io/tyk-gateway/tyk-gateway:{{ .Tag }}"
+  build_flag_templates:
+    - "--build-arg=PORTS=8080"
+    - "--platform=linux/amd64"
+    - "--label=org.opencontainers.image.created={{.Date}}"
+    - "--label=org.opencontainers.image.title={{.ProjectName}}"
+    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+    - "--label=org.opencontainers.image.version={{.Version}}"
+  use: buildx
+  goarch: amd64
+  goos: linux
+  dockerfile: ci/Dockerfile.std
+  extra_files:
+    - "ci/install/"
+    - "README.md"
+    - "LICENSE.md"
+    - "apps/app_sample.json"
+    - "templates"
+    - "middleware"
+    - "event_handlers/sample"
+    - "policies"
+    - "coprocess"
+    - "tyk.conf.example"
+
+# Build gateway hybrid container amd64
+- ids:
+    - std
+  image_templates:
+    - "tykio/tyk-hybrid-docker:{{ .Tag }}-amd64"
+  build_flag_templates:
+    - "--platform=linux/amd64"
+    - "--label=org.opencontainers.image.created={{.Date}}"
+    - "--label=org.opencontainers.image.title={{.ProjectName}}-hybrid"
+    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+    - "--label=org.opencontainers.image.version={{.Version}}"
+  use: buildx
+  goarch: amd64
+  goos: linux
+  dockerfile: ci/images/hybrid/Dockerfile
+  extra_files:
+    - "ci/images/hybrid/"
+# Build tykio/tyk-gateway, docker.tyk.io/tyk-gateway/tyk-gateway (arm64)
+- ids:
+    - std
+  image_templates:
+    - "tykio/tyk-gateway:{{ .Tag }}-arm64"
+    - "docker.tyk.io/tyk-gateway/tyk-gateway:{{ .Tag }}-arm64"
+  build_flag_templates:
+    - "--build-arg=PORTS=8080"
+    - "--platform=linux/arm64"
+    - "--label=org.opencontainers.image.created={{.Date}}"
+    - "--label=org.opencontainers.image.title={{.ProjectName}}"
+    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+    - "--label=org.opencontainers.image.version={{.Version}}"
+  use: buildx
+  goarch: arm64
+  goos: linux
+  dockerfile: ci/Dockerfile.std
+  extra_files:
+    - "ci/install/"
+    - "README.md"
+    - "LICENSE.md"
+    - "apps/app_sample.json"
+    - "templates"
+    - "middleware"
+    - "event_handlers/sample"
+    - "policies"
+    - "coprocess"
+    - "tyk.conf.example"
+
+# Build gateway hybrid container arm64
+- ids:
+    - std
+  image_templates:
+    - "tykio/tyk-hybrid-docker:{{ .Tag }}-arm64"
+  build_flag_templates:
+    - "--platform=linux/arm64"
+    - "--label=org.opencontainers.image.created={{.Date}}"
+    - "--label=org.opencontainers.image.title={{.ProjectName}}-hybrid"
+    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+    - "--label=org.opencontainers.image.version={{.Version}}"
+  use: buildx
+  goarch: arm64
+  goos: linux
+  dockerfile: ci/images/hybrid/Dockerfile
+  extra_files:
+    - "ci/images/hybrid/"
+
+# Build plugin-compiler
+- ids:
+    - std
+  image_templates:
+    - tykio/tyk-plugin-compiler:{{ .Tag }}
```

Comment:
It is probably not necessary to provide a separate plugin-compiler as there is no difference in the image?